CREDITS

```diff
@@ -79,7 +79,7 @@ research project called The Harvest Information Discovery and Access System:
  *   the technology.  The Harvest software is really a reference
  *   implementation of a set of protocols and formats, some of which
  *   we intend to standardize.  We encourage commercial
- *   re-implementations of code complying to this set of standards.  
+ *   re-implementations of code complying to this set of standards.
```

Comment:
Please undo the out-of-scope whitespace changes above (and further below). There should be no CREDITS changes except for GnuRegex blobs removal.

Reply:
Done. Editor is set to remove white space at end of line and I didn't notice.
Maybe we could clear it in a separate PR and make it part of source-maintenance.sh?

Reply:
> Editor is set to remove white space at end of line and I didn't notice.

I know. We all have been there. FWIW, I recommend running a git diff against the branch forking point _before_ posting a PR _and_ looking at the GitHub-generated diff afterwards. I have caught lots of problems following this simple routine.

>  Maybe we could clear it in a separate PR

That would be nice, especially if you apply that fix to `git grep '\s$'` (with exceptions for RFCs and such) rather than a single file.

> and make it part of source-maintenance.sh?

It already is a part of test-suite/test-sources.sh (that automatically runs on GitHib Actions). Look for `diff --check` there. We should start to enforcing those rules soon (the check failures are currently reported but otherwise ignored).